mk_verificator/utils/ssh_client.py

```diff
@@ -0,0 +1,181 @@
+        ssh.set_missing_host_key_policy(AutoAddPolicy())
+
+        if self.pub_key:
+            pkey = RSAKey.from_private_key_file(self.pub_key)
+            ssh.connect(hostname=self.hostname, username=self.username,
+                        pkey=pkey, look_for_keys=True)
+        elif self.password:
+            ssh.connect(hostname=self.hostname, port=self.ssh_port,
+                        username=self.username, password=self.password,
+                        look_for_keys=True)
+        else:
+            raise ValueError('password or pub_key is required')
+
+        return ssh
+
+    def close(self):
+        self.ssh.close()
+
+    def run(self, command, sudo=False, timeout=None):
+        """
+
+        :param command: - string with bash/cli command
+        :param sudo: - boolean
+        :param timeout: integer value - command timeout
+        :return:
+        """
+        if timeout:
+            command = "timeout --signal=9 {0} {1}".format(timeout, command)
+        if sudo:
+            command = "sudo {0}".format(command)
+
+        msg = "{0} ({1}): {2}".format(self.id, self.hostname, command)
+        self.logger.debug(msg)
+
+        try:
+            stdin, stdout, stderr = self.ssh.exec_command(command)
+        except AttributeError as NoOpenSession:
+            msg = "SSH Manager: no conn for {0}".format(self.id)
+            self.logger.error(msg)
+            raise NoOpenSession
+
+        out = stdout.read().strip('\n')
+        err = stderr.read().strip('\n')
+        result = err if err else out
+        self.logger.debug(result)
+
+        return result
+
+    def run_channel_command(self, command):
+        client = self.ssh
+        try:
+            channel = client.get_transport().open_session()
+            channel.exec_command(command)
+            output = ''
+            while True:
+                if channel.exit_status_ready():
+                    output = output + channel.recv(1048576)
+                    break
+                rl, wl, xl = select.select([channel], [], [], 0.0)
+                if len(rl) > 0:
+                    output = output + channel.recv(1024)
+            return output
+        except Exception as e:
+            self.logger.error(e)
+        finally:
+            client.close()
+
+    def get(self, remotepath, localpath):
+        self.logger.debug("sftp get %s %s", remotepath, localpath)
+        with SFTPClient.from_transport(self.ssh.get_transport()) as sftp:
+            sftp.get(remotepath=remotepath, localpath=localpath)
+
+    def put(self, localpath, remotepath):
+        self.logger.debug("sftp put %s %s", localpath, remotepath)
+        with SFTPClient.from_transport(self.ssh.get_transport()) as sftp:
+            sftp.put(localpath=localpath, remotepath=remotepath)
+
+    def rm(self, path):
+        command = "rm {0}".format(path)
+        return self.run(command, sudo=True)
+
+    def tcpdump(self, iface, filters=None, count=0, timeout=0, clear=True):
```

Comment:
as was discussed, for now we have no major reason to keep it within our repo, then let's delete it because of additional external requirement